src/constants/CODATA2018/test/CODATA2018.test.cpp

```diff
@@ -60,17 +60,18 @@ void checkMap( MAP& map ){
       auto verifyIfExists = [&](auto key) {
         return hana::overload(
           []( hana::true_ ){ return true; },
-          [&]( auto value ){ return reference.second == value.value; } 
+          [&]( auto value ){ return reference.second == value.value; }
         )( hana::find( map.uncertainty, key ).value_or( hana::true_c ) );
       };
       auto verifyMatchingUnits = [&](auto key, auto units) {
         return hana::overload(
           []( hana::true_ ){ return true; },
-          [&]( auto value ){ return value.units() == units; } 
+          [&]( auto value ){ return value.units() == units; }
         )( hana::find( map.uncertainty, key ).value_or( hana::true_c ) );
       };
       CHECK( fabs( 1 - (reference.first/map[ key ].value ) ) < 1E-10 );
       CHECK( verifyIfExists( key ) );
+      (void)verifyMatchingUnits; // FIXME silence "unused variable" warning
```

Comment:
I had added the lambda, `verifyMatchingUnits`, to ensure that the constant and the uncertainty had the same units. Unfortunately, I actually never used it. We should probably get rid of the lambda instead of silencing the warnings as you have done.

Reply:
OK, I got rid of it.

By the way, I noticed a bit of seemingly inconsistent terminology between the 2014 and 2018 data:

   1. For electric: "electric constant" vs. "vacuum electric permittivity"

   2. For magnetic: "mag. constant" vs. "vacuum mag. permeability"

   3. For reducedPlanck and hbar: "Planck constant over 2 pi" vs. "reduced Planck constant"

Also, the 2014 "test all the constants" scenario includes hbar, while the 2018 one does not.

Are these things worth worrying about? (Note: I'm known for being overly pedantic sometimes. :-) But it's arguably a good thing for a software person.)

Reply:
The strings you quote in 1–3 are the strings that are found in the reference file that was downloaded from NIST. They are used to identify the constants found in the reference file and, thus, must be different if the reference is different.

We should probably add hbar in 2018.